<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="viewAll">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View All Products</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateProduct">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addProduct">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProduct">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Management Interface</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
